samples/storage_csm_powerstore_v250.yaml

```diff
@@ -0,0 +1,139 @@
+apiVersion: storage.dell.com/v1
+kind: ContainerStorageModule
+metadata:
+  name: test-powerstore
+  namespace: test-powerstore
+spec:
+  driver:
+    csiDriverType: "powerstore"
+    csiDriverSpec:
+      # fsGroupPolicy: Defines if the underlying volume supports changing ownership and permission of the volume before being mounted.
+      # Allowed values: ReadWriteOnceWithFSType, File , None
+      # Default value: ReadWriteOnceWithFSType
+      fSGroupPolicy: "File"
```

Comment:
@kk3, please ensure that changes made in samples are reflected properly as i see there is hardcoding currently